src/Revit/RevitHyparTools/Create.Wall.cs

```diff
@@ -29,13 +26,20 @@ public static Elements.Wall[] WallsFromRevitWall(ADSK.Wall wall, Document doc)
             }
 
             var wallPlane = wallFace as PlanarFace;
-            var profiles = GetProfilesOfFace(wallPlane);
+            var profiles = wallPlane.GetProfiles();
 
             var centerline = (wall.Location as LocationCurve).Curve;
+
             var line = new ElemGeom.Line(centerline.GetEndPoint(0).ToVector3(), centerline.GetEndPoint(1).ToVector3());
```

Comment:
Can we just multiply the vectors here by the scale? Do we need the additional transform steps below? 
`var line = new ElemGeom.Line(centerline.GetEndPoint(0).ToVector3() * Units.FeetToMeters(1),...`

Reply:
I suggest this because matrix operations are expensive. We should avoid them where we can.

Reply:
okay, done.